src/main/java/com/dataloom/datastore/services/SearchService.java

```diff
@@ -442,10 +464,10 @@ public SearchResult executeFQNPropertyTypeSearch( String namespace, String name,
 
     }
 
-    private boolean getAuthorization( UUID entitySetId ) {
-        return authorizations.checkIfHasPermissions( new AclKey( entitySetId ),
-                Principals.getCurrentPrincipals(),
-                EnumSet.of( Permission.READ ) );
+    private boolean getAuthorization( UUID entitySetId, Set<Principal> principals ) {
```

Comment:
You should implement AuthorizingComponent eventually, we should have an optimized and correct version of authz code that is centralized to make it easier to maintain.